Taboola SDK Swift Sample/Taboola SDK Swift Sample/SDK Standard/UITableView Constraints/CustomTaboolaView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CustomTaboolaView.swift
+//  Taboola SDK Swift Sample
+//
+//  Created by Liad Elidan on 24/11/2019.
+//  Copyright © 2019 Taboola LTD. All rights reserved.
+//
+
+import Foundation
```

Comment:
You don't need to import Foundation when using UIKit.

Reply:
fixed